### PR TITLE
Show larva position on button

### PR DESCRIPTION
**Pull request by SandPoot**
## About The Pull Request
Title says all you need to know
image
## Why It's Good For The Game
Larb
## Changelog
:cl:
qol: The larva queue button shows your position when active.
/:cl:

### CODE REVIEW

File: code/datums/components/larva_queue.dm [hunk 1]
```diff
@@ -91,9 +91,13 @@
 /datum/component/larva_queue/proc/set_queue_position(waiter, new_position)
 	SIGNAL_HANDLER
 	position = new_position
+	var/datum/hive_status/the_hive = GLOB.hive_datums[XENO_HIVE_NORMAL]
```
Comment (Lumipharon) [inline code/datums/components/larva_queue.dm:94]: isn't this going to runtime if no hive is setup?

File: code/datums/components/larva_queue.dm [hunk 2]
```diff
@@ -91,9 +91,13 @@
...
+	action.button.maptext = MAPTEXT_TINY_UNICODE("<span class='center'>[position]/[LAZYLEN(the_hive.candidates)]</span>")
```
Comment (Lumipharon) [inline code/datums/components/larva_queue.dm:95]: use span macros

**Review by Lumipharon — CHANGES_REQUESTED** [review]

File: code/datums/components/larva_queue.dm (hunk 1 above)
Comment (SandPoot) [reply code/datums/components/larva_queue.dm:94]: Aren't there going to be normal hives always anyways? and if this is true, this means that the button will runtime every click at line (previously 113) 117.

Comment (Lumipharon) [reply code/datums/components/larva_queue.dm:94]: I unno, its mainly for before the game starts, but I wouldn't just trust existing code to believe whether it runtimes lol

File: code/datums/components/larva_queue.dm (hunk 2 above)
Comment (Lumipharon) [reply code/datums/components/larva_queue.dm:95]: if there isn't one already, yeah

**Comment by Lumipharon** [comment]
No these span macros 
image

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**